common/changes/@itwin/core-backend/Soham-extendedTypeFixConcurrentQuery_2024-11-13-14-06.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Added test case to document the behaviour of properties extendType and extendedType of QueryPropertyMetaData",
```

```suggestion
      "comment": "",
```

End users don't care about tests.